src/webhooks/pubsub/slackScores.ts

```diff
@@ -18,101 +18,100 @@ const NUM_ROW_SPACES = 3;
 const TEAM_PREFIX = 'team-';
 
 export const triggerSlackScores = async (
-  __org: GitHubOrg,
+  org: GitHubOrg,
   __now: moment.Moment
 ) => {
-  const teamScores: teamScoreInfo[] = await Promise.all(
-    Object.keys(PRODUCT_OWNERS_INFO['teams']).map(async (team: string) => {
-      // Filter out issues that are not yet due
-      const issueTriageEvents = (await getIssueEventsForTeam(team)).filter(
-        (issue) => moment(issue.triage_by_dt.value) <= moment()
-      );
-      const triagedOnTimeEvents = issueTriageEvents.filter(
-        (issue) => issue.triaged_dt.value <= issue.triage_by_dt.value
-      );
-      // For teams with 0 events, set the score to 0 instead of NaN to ensure those teams
-      // end up on the bottom of the scoreboard automatically.
-      const score =
-        issueTriageEvents.length === 0
-          ? 0
-          : triagedOnTimeEvents.length / issueTriageEvents.length;
-      return {
-        team: team.slice(TEAM_PREFIX.length),
-        score,
-        eventsTriagedOnTime: triagedOnTimeEvents.length,
-        numEvents: issueTriageEvents.length,
-      };
-    })
-  );
+  if (org.slug === 'getsentry') {
```

Comment:
Moving the code to the right like this feels unnecessary. Its best practice to return early on exit conditions like this, to avoid this sort of nesting:
```
if (org.slug != 'getsentry') {
  return
}

// ... rest of your logic...
```

Will also make the diff a lot smaller too!

Reply:
done.